webapp/src/main/java/com/box/l10n/mojito/android/strings/AndroidStringDocument.java

```diff
@@ -51,7 +52,7 @@ public void addPlural(AndroidStringElement element, Node comment) {
 
         AndroidPlural.AndroidPluralBuilder builder = AndroidPlural.builder();
         builder.setName(element.getNameAttribute());
-        builder.setComment(comment.getTextContent());
+        Optional.ofNullable(comment).ifPresent(c -> builder.setComment(c.getTextContent()));
```

Comment:
same here, why not just using a "if" statement within that function?

Reply:
same